build/charts/antrea/crds/bgppolicy.yaml

```diff
@@ -2,12 +2,15 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: bgppolicies.crd.antrea.io
+  labels:
+    app: antrea
+    served-by: antrea-controller
 spec:
   group: crd.antrea.io
   versions:
     - name: v1alpha1
       served: true
-      storage: true
```

Comment:
need to configure:
```
  conversion:
    strategy: Webhook
    webhook:
      conversionReviewVersions: ["v1", "v1beta1"]
      clientConfig:
        service:
          name: "antrea"
          namespace: "kube-system"
          path: "/convert/bgppolicy"
```
Otherwise the webhook won't be working